src/client/Microsoft.Identity.Client/Platforms/Features/WamBroker/WamBroker.cs

```diff
@@ -73,9 +74,12 @@ internal class WamBroker : IBroker
             _aadPlugin = testAadPlugin ?? new AadPlugin(_wamProxy, _webAccountProviderFactory, _logger);
             _msaPlugin = testmsaPlugin ?? new MsaPlugin(_wamProxy, _webAccountProviderFactory, _logger);
 
-            _msaPassthroughHandler = msaPassthroughHandler ?? 
+            _msaPassthroughHandler = msaPassthroughHandler ??
                 new MsaPassthroughHandler(_logger, _msaPlugin, _wamProxy, _parentHandle);
 
+            _wamOptions = appConfig.WindowsBrokerOptions ?? 
+                new WindowsBrokerOptions();
```

Comment:
nit: What it be appropriate to call `WindowsBrokerOptions.CreateDefault()` here?